blockchain/state/statedb_test.go

```diff
@@ -322,7 +322,7 @@ func TestCachedStateObjects(t *testing.T) {
 // it should be loaded from underlying state trie or persistent database.
 func TestCachedStateObjectsEviction(t *testing.T) {
 	stateDB, _ := New(common.Hash{}, NewDatabase(database.NewMemoryDBManager()))
-	stateDB.cachedStateObjects = common.NewCache(common.LRUConfig{CacheSize: 1})
+	stateDB.cachedStateObjects = common.NewCache(common.LRUConfig{CacheSize: 1, IsScaled: true})
```

Comment:
I think we should not scale the cache in testing.
During this test, I think the cache size is zero, So the cache doesn't work properly.

Please consider remote `IsScaled:true` in all test cases except the test for the scaling logic.

Reply:
I took your suggestion and changed to not scaling, along with other 3 tests.